hw/ip/pattgen/doc/_index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "PATTGEN HWIP Technical Specification"
+---
+
+# Overview 
+
+This document specifies PATTGEN hardware IP (HWIP) functionality.
+This module conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification/index.md" >}})
+See that document for integration overview within the broader top-level system.
+
+## Features
+
+- Generates time-dependent patterns on two (2) channels, each with its own clock.
+-    In each channel, data is transmitted serially on a one-bit data (pda) output, which is synchronous to a corresponding parallel clock signal (pcl).
+   -    The channels may operate independently or synchronously with each other. 
```

Comment:
Are the two bullet points above meant to be under the first bullet point? If so, indent the `-`.

Reply:
Done. Thanks.